cli/src/cmd/testnet/start.rs

```diff
@@ -110,62 +113,62 @@ impl TestnetStartCmd {
-        println!("\n⚙️  Generating genesis file...");
+        info!("Generating genesis file");
         self.generate_genesis(home_dir)?;
-        println!("✓ Genesis file created");
+        info!("Genesis file created");
 
         // 5. Spawn Reth processes
-        println!("\n🔗 Starting Reth execution clients...");
+        info!("Starting Reth execution clients");
         let reth_processes = self.spawn_reth_nodes(home_dir)?;
-        println!("✓ All Reth nodes started");
+        info!("All Reth nodes started");
 
         // 6. Wait for Reth nodes to be ready
-        println!("\n⏳ Waiting for Reth nodes to initialize...");
+        info!("Waiting for Reth nodes to initialize");
         self.wait_for_reth_nodes(home_dir)?;
-        println!("✓ All Reth nodes ready");
+        info!("All Reth nodes ready");
 
         // 7. Connect Reth peers
-        println!("\n🔗 Connecting Reth peers...");
+        info!("Connecting Reth peers");
         self.connect_reth_peers(home_dir)?;
-        println!("✓ Reth peers connected");
+        info!("Reth peers connected");
 
         // 8. Spawn Emerald processes
-        println!("\n💎 Starting Emerald consensus nodes...");
+        info!("Starting Emerald consensus nodes");
         let emerald_processes = self.spawn_emerald_nodes(home_dir)?;
-        println!("✓ All Emerald nodes started");
-
-        println!("\n✅ Testnet started successfully!");
-        println!("\n📊 Status:");
-        println!("  Reth processes: {} running", reth_processes.len());
-        println!("  Emerald processes: {} running", emerald_processes.len());
-        println!("\n📁 Logs:");
-        println!(
+        info!("All Emerald nodes started");
+
+        info!("Testnet started successfully!");
+        info!("Status:");
+        info!("  Reth processes: {} running", reth_processes.len());
+        info!("  Emerald processes: {} running", emerald_processes.len());
+        info!("Logs:");
+        info!(
             "  Reth: {}/{{0..{}}}/logs/reth.log",
             home_dir.display(),
             self.nodes - 1
         );
-        println!(
+        info!(
             "  Emerald: {}/{{0..{}}}/logs/emerald.log",
             home_dir.display(),
             self.nodes - 1
         );
 
-        println!("\n💡 Commands:");
-        println!("    emerald testnet status           - Check status of all nodes");
-        println!("    emerald testnet stop-node <id>   - Stop a specific node");
-        println!("    emerald testnet stop             - Stop all nodes");
-        println!("    emerald testnet destroy          - Remove all testnet data");
+        info!("Commands:");
+        info!("    emerald testnet status           - Check status of all nodes");
+        info!("    emerald testnet stop-node <id>   - Stop a specific node");
+        info!("    emerald testnet stop             - Stop all nodes");
+        info!("    emerald testnet destroy          - Remove all testnet data");
```

Comment:
For this one, since it is an example of commands we could maybe use line returns `\n` to have it as a 1 liner while keeping the format